tests/8.LogicSig.js

```diff
@@ -321,12 +321,29 @@ describe('Logic validation', () => {
       result = logic.checkProgram(program);
       assert.equal(result, true);
 
-      // 800x keccak256 more is to costly
+      // 800x keccak256 more is too costly
       program = utils.concatArrays(program, new Uint8Array(800).fill(2));
-      assert.throws(
-        () => logic.checkProgram(program),
-        new Error('program too costly to run')
-      );
+      //  old versions
+      const oldVersions = [0x1, 0x2, 0x3];
+      let v;
+      for (v of oldVersions) {
+        program[0] = v;
+        assert.throws(
+          () => logic.checkProgram(program),
+          new Error('program too costly to run')
```

Comment:
shouldn't the error message have been updated to 'program too costly for Teal version < 4. consider using v4.'?